src/app/onboarding/legal-fellow-credentials/page.tsx

```diff
@@ -42,17 +36,19 @@ export default function Page() {
   useScrollToTop();
 
   const [expectedBarDate, setExpectedBarDate] = useState<string>(
-    onboarding.profile.expected_bar_date ?? '',
+    onboarding.profile.expected_bar_date
+      ? parseDateAlt(onboarding.profile.expected_bar_date)
+      : '',
   );
 
   // initialize form with values from onboarding context
-  const form = useForm<z.infer<typeof legalExperienceSchema>>({
-    resolver: zodResolver(legalExperienceSchema),
+  const form = useForm<z.infer<typeof legalFellowCredentialSchema>>({
+    resolver: zodResolver(legalFellowCredentialSchema),
     defaultValues: {
       expectedBarDate: onboarding.profile.expected_bar_date
-        ? new Date(`${onboarding.profile.expected_bar_date}T00:00`)
+        ? onboarding.profile.expected_bar_date
```

Comment:
was there a schema change that warranted this change?

Reply:
ah yes, recall the PR description:
> Added ProfileToUpload schema type that extends the regular Profile but also accepts null values for optional values (used to upsert NULL into Supabase)

I forgot to add that this also included changing timestamp `string` into just JS `Date` objects, which was more compatible with `timestamptz` type on Supabase.